### DISCUSSION TITLE
Creating new roles

### DISCUSSION THREAD

**Discussion by Pietro-Porsch-W** [chatwoot/chatwoot#10081] [Feature Requests]
Just more roles to use, or could give us a way to create new roles to give permission to use some things that Agents don't have access to.

For example create an role to the user that he can alter the help center, or can create automations, etc.

This suggestions is because only an administrator, who have access to delete and create anything, can do it, soo i do it, or i give someone more access than necessary to do it, which is not a secure way to work.

**Comment by sojan-official** [comment]
This will be handled in the upcoming permissions feature : https://github.com/chatwoot/chatwoot/issues/4216
